### PR TITLE
Fixes #9913: Add SELinux config to relay package

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/9913

### CODE REVIEW

**Comment by amousset** [comment]
Do not merge for now, missing .te file

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
Still not complete

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
ping @peckpeck

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
OK, merging this PR

File: rudder-webapp/SOURCES/rudder-webapp.fc [hunk 1]
```diff
@@ -1,3 +1,5 @@
 /var/rudder/inventories/accepted-nodes-updates gen_context(system_u:object_r:httpd_sys_rw_content_t,s0)
 /var/rudder/inventories/incoming               gen_context(system_u:object_r:httpd_sys_rw_content_t,s0)
 /var/log/rudder/apache2(/.*)?                  gen_context(system_u:object_r:httpd_log_t,s0)
+/opt/rudder/etc/uuid.hive                      gen_context(system_u:object_r:httpd_sys_content_t,s0)
```
Comment (ncharles) [inline rudder-webapp/SOURCES/rudder-webapp.fc]: shouldn't it be removed in 4.1 where relay is on server ?

**Pull request merged**